src/lib/rest/rest.cpp

```diff
@@ -1239,6 +1245,23 @@ static int connectionTreat
       return MHD_NO;
     }
 
+
+    // Get API version
+    ciP->apiVersion = (url[2] == '2')? V2 : V1;  // If an APIv2 request, the URL starts with "/v2/". Only V2 requests.
```

Comment:
So... /m2/version should be interpreted also as API v2? ;)

I'd suggest to add a check for 'v' in url[1] to this.

Reply:
The check is valid.
What you say is valid too, but still doesn't cover "/v23", etc ...
If we ever invent a REST service with the third char '2' and its not V2 ...
Well, then everything will break and we will notice at once.

That said, in a nextcoming PR this 'if' is removed (I found a function thrther down in the code, and that function is used instead.

So, NTC, until next PR is launched

Reply:
In order to ease the review of that new PR, a FIXME mark should be introduced. Something like:

```
FIXME #3109-PR: this if will be removed and will use function X
```

Even with my legendary memory ;) it is hard when you review a PR remember comments on a past one.

Reply:
Fixed in 1e2e62c